test/gleam/test/a/src/dummy.gleam

```diff
@@ -0,0 +1,9 @@
+import gleam/io
```

Comment:
remove file, not needed

Reply:
[eb0f485](https://github.com/containerbase/base/pull/2018/commits/eb0f485c857bdb4c9a99270730d5ca9d17c9d706)